index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="resume">
+        <h1 class="resume__text">Resume</h1>
+      </div>
+
+      <div class="header photo">
+        <img
+          class="photo__img"
+          src="./img/MyPhoto.png"
+          alt="MyPhoto"
+          width="300px"
+          height="300px"
+        />

```

Comment:
all attributes of the image tag shoul be in one line. These line breaks are validation error